Examples/ObjectiveC/ShapeCollectionFeatureExample.m

```diff
@@ -0,0 +1,56 @@
+#import "ShapeCollectionFeatureExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleShapeCollectionFeature = @"ShapeCollectionFeatureExample";
+
+@interface ShapeCollectionFeatureExample () <MGLMapViewDelegate>
+
+@end
+
+@implementation ShapeCollectionFeatureExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame: self.view.bounds styleURL: [MGLStyle lightStyleURLWithVersion:9]];
+    mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    
+    [mapView setCenterCoordinate:CLLocationCoordinate2DMake(38.897435, -77.039679) zoomLevel:12 animated:NO];
+    mapView.delegate = self;
+    
+    [self.view addSubview:mapView];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    
+    NSURL *url = [[NSURL alloc] initWithString: @"https://api.mapbox.com/datasets/v1/mapbox/cj004g2ay04vj2xls3oqdu2ou/features?access_token=pk.eyJ1IjoibWFwYm94IiwiYSI6ImNpemc0YWlpNzAwcXUyd21ldDV6OWpxMGwifQ.A92RQZpwUgtGtCmdSE4-ow"];
+    if (url) {
+        NSData *data = [[NSData alloc] initWithContentsOfURL:url];
+        MGLShape *feature = [MGLShape shapeWithData:data encoding: NSUTF8StringEncoding error: nil];
+        
+        // Create source and add it to the map style.
+        MGLShapeSource *source = [[MGLShapeSource alloc] initWithIdentifier:@"transit" shape:feature options:nil];
```

Comment:
Since you don’t need to manipulate the GeoJSON data before creating a shape source with it, you could instead call `-[MGLShapeSource initWithIdentifier:URL:options:]`. There are strong use cases for either approach, so use the approach you think is more important to teach in this example.